### DISCUSSION TITLE
Is validation results in summary.json performed in the whole validation volume or patches?

### DISCUSSION THREAD

**Discussion by dscarmo** [MIC-DKFZ/nnUNet#978] [Q&A] [answered]
Hello, I have a question.

The summary.json file has results for every validation subject. Are these calculated with the whole original subject and provided target? Or some kind of cropping? 

I suppose so, but I just want to be sure.

Thank you for providing this amazing resource.

**Comment by FabianIsensee** [comment, marked as answer]
they are all computed in the original geometry. So the predicted segmentations are resampled to the original resolution and any cropping has been reversed
